tools/flake8-torch/flake8_torch/checker.py

```diff
@@ -42,27 +102,110 @@ class TorchChecker:
+        module.visit(visitor)
+        self.violations = visitor.violations
 
     def run(self):
-        yield from self._check_deprecated_functions()
+        for violation in self.violations:
+            yield violation.flake8_result()
+
+
+class TorchCodemod(codemod.Codemod):
+    def transform_module_impl(self, module: cst.Module) -> cst.Module:
+        visitor = TorchVisitor(_read_deprecated_config())
+        fixes_count = 0
+
+        # We use `unsafe_skip_copy`` here not only to save some time, but
+        # because `deep_replace`` is identity-based and will not work on
+        # the original module if the wrapper does a deep copy:
+        # in that case we would need to use `wrapped_module.module`
+        # instead of `module`.
+        wrapped_module = cst.MetadataWrapper(module, unsafe_skip_copy=True)
+        wrapped_module.visit(visitor)
+        new_module = module
+
+        violations = visitor.violations
+        for violation in violations:
+            if violation.replacement is not None:
+                new_module = new_module.deep_replace(
+                    violation.node, violation.replacement
+                )
+                fixes_count += 1
+            try:
+                path = Path(self.context.filename).relative_to(Path.cwd())
+            except ValueError:
+                # Not a subpath of a current dir, use absolute path
+                path = self.context.filename
+            print(f"{path}{violation.codemod_result()}")
+        if fixes_count == 0:
+            raise codemod.SkipFile("No changes")
+        return new_module
+
+
+def main():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "path",
+        nargs="+",
+        help=("Path to check/fix. Can be a directory, a file, or multiple of either."),
+    )
+    parser.add_argument(
+        "--fix",
+        action="store_true",
+        help="Fix fixable violations.",
+    )
+    parser.add_argument(
+        "-j",
+        "--jobs",
+        help="Number of jobs to use when processing files. Defaults to number of cores",
+        type=int,
+        default=None,
+    )
+
+    args = parser.parse_args()
+
+    files = codemod.gather_files(args.path)
+    command_instance = TorchCodemod(codemod.CodemodContext())
+    DIFF_CONTEXT = 5
+    try:
+        # Silence "Failed to determine module name"
+        # https://github.com/Instagram/LibCST/issues/944
+        with contextlib.redirect_stderr(io.StringIO()):
+            result = codemod.parallel_exec_transform_with_prettyprint(
+                command_instance,
+                files,
+                jobs=args.jobs,
+                unified_diff=(None if args.fix else DIFF_CONTEXT),
+                hide_progress=True,
+                format_code=False,
+                repo_root=None,
+            )
+    except KeyboardInterrupt:
+        print("Interrupted!", file=sys.stderr)
+        return 2
```

Comment:
`main` return code is ignored, isn't it?
```suggestion
        sys.exit(2)
```

Reply:
Right, I'll fix all of these.